packages/cozy-harvest-lib/src/datacards/makeViewerFiles.js

```diff
@@ -0,0 +1,83 @@
+import { useMemo } from 'react'
+import get from 'lodash/get'
+import sortBy from 'lodash/sortBy'
+import uniq from 'lodash/uniq'
+
+import CozyClient, { Q, useQuery } from 'cozy-client'
+
+const useFolderToSave = triggerId => {
+  const { data, fetchStatus } = useQuery(
+    Q('io.cozy.triggers').getById(triggerId),
+    { as: `io.cozy.triggers/${triggerId}` }
+  )
+
+  return fetchStatus === 'loaded' ? data?.[0]?.message?.folder_to_save : null
+}
+
+const handleReturn = ({ data, fetchStatus }) =>
+  fetchStatus === 'loaded' && data.length === 0
+    ? []
+    : fetchStatus !== 'pending' && data.length > 0
+    ? data
+    : null
```

Comment:
I don't think we need this method. Cozy-Client already return an empty array if `data.length === 0`. 

I prefer if we avoid to edit the return of cozy-client. 

What is the gain to return null? Why not dealing with fetchStatus within the component?

Reply:
- [ ] Will use `type QueryFetchStatus = "failed" | "loading" | "pending" | "loaded";` along with eventual data